package/Dockerfile

```diff
@@ -1,14 +1,5 @@
 FROM alpine:3.12
 
-MAINTAINER Frank Mai <frank@rancher.com>
-
-RUN apk add --no-cache --update \
```

Comment:
Why did we remove this step?

Reply:
I was trying to root cause the build failure and removed that since it doesn't appear in other repositories, but that doesn't seem to be the issue. Will add it back in